src/google/protobuf/arena.h

```diff
@@ -348,7 +348,7 @@ class PROTOBUF_EXPORT PROTOBUF_ALIGNAS(8) Arena final {
     }
   }
 
-  // The following are routines are for monitoring. They will aproximate the
```

Comment:
CHANGES.txt

Reply:
Are you suggesting that something needs to be added to `CHANGES.txt` here?